src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpWindowsProxy.cs

```diff
@@ -166,6 +179,21 @@ private HttpWindowsProxy(WinInetProxyHelper proxyHelper, SafeWinHttpHandle sessi
                     }
                 }
             }
+
+            _cachedProxies = new ConcurrentDictionary<string, CachedProxy>();
+            _cacheCleanupTimer = new Timer(state =>
+            {
+                var cachedProxies = (ConcurrentDictionary<string, CachedProxy>)state;
+                int ticks = Environment.TickCount;
```

Comment:
The chances of this being a problem are minuscule, but to be on the safe side it'd probably be better to use longs and TickCount64.

Reply:
If the timer is running every 30 seconds, this shouldn't be an issue. Are you worried about the timer running ~40 days late?

Reply:
Timer removed.